openlibrary/macros/FormatExpiry.html

```diff
@@ -4,21 +4,21 @@
     $if render_once('expire-dates-js'):
         <script type="text/javascript">
         <!--
-        window.q.push(function(){
-            \$(".localize-time").each(function() {
-                var expiry_utc = \$(this).attr("data-expiry-utc");
-                var options = {hour: '2-digit', minute:'2-digit', month:'2-digit', day:'2-digit', year:'numeric'};
-                \$(this).html((new Date(expiry_utc)).toLocaleString([], options));
-            });
-        });
+        // window.q.push(function(){
+        //     \$(".localize-time").each(function() {
+        //         var expiry_utc = \$(this).attr("datetime");
+        //         var options = {hour: '2-digit', minute:'2-digit', month:'2-digit', day:'2-digit', year:'numeric'};
+        //         \$(this).html((new Date(expiry_utc)).toLocaleString([], options));
+        //     });
+        // });
         //-->
         </script>
 
     $ expiry_datetime = datetime_from_isoformat(expiry)
     <div class="local-date-time">
       $_('Expires')
-      <span class="localize-time"
-	    data-expiry-utc="$datetimestr_utc(expiry_datetime)">
-	$datetimestr_utc(expiry_datetime)
-      </span>
+      <time class="localize-time"
+	    datetime="$datestr(expiry_datetime)">
```

Comment:
The hidden value should be the UTC time here.

```suggestion
	    datetime="$datetimestr_utc(expiry_datetime)"
            title="$datetimestr_utc(expiry_datetime)">
```